pymuonsuite/io/uep.py

```diff
@@ -124,46 +213,7 @@ def read(self):
             self._Etot = results['Etot']
             self._x_opt = results['x']
             self._fx_opt = results['fx']
+            self.atoms = results['struct']
```

Comment:
Any good reasons for not making this a hidden `@property` like everything else is?

Reply:
I did this because the UEP calculator already has a property called 'atoms' and I thought it would make sense to read the results into this property. I will think some more about whether this is the correct thing to do...

Reply:
Yes, I believe the right thing to do is to read the structure into the calculator's atoms property, which is what is currently being done

Reply:
With regards to why the atoms property isn't a hidden property anyway, I think the distinction between them is that 'atoms' is a property that all the ASE calculators would have, and you would expect to access it by calling `calc.atoms` (if calc was your instance of the calculator)

Reply:
Ah, fair, yes. Sorry if I brought this up, makes sense.

Reply:
No problem!